core/src/test/java/org/testcontainers/DockerRegistryContainer.java

```diff
@@ -0,0 +1,68 @@
+package org.testcontainers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.SneakyThrows;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.FrameConsumerResultCallback;
+import org.testcontainers.containers.output.OutputFrame;
+import org.testcontainers.containers.output.WaitingConsumer;
+
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DockerRegistryContainer extends GenericContainer<DockerRegistryContainer> {
```

Comment:
Given that this is a new public container implementation that is available to users, should we also add it to the docs? (can be follow-up PR).

Reply:
it is in `core/src/test/` :)

Reply:
Whoops, you are right. Probably also too specific for regular users.